Util/pose.py

```diff
@@ -6,10 +6,10 @@
 
 ORIENTATION_ABSOLUTE_TOLERANCE = 0.004
 
-
+# pylint: disable=invalid-name
 class Pose:
 
-    def __init__(self, position: Position=Position(), orientation: float=0):
+    def __init__(self, position: Position = Position(), orientation: float = 0):
```

Comment:
Actually, I think PEP8 is without space. [«Don't use spaces around the = sign when used to indicate a keyword argument or a default parameter value.»](https://www.python.org/dev/peps/pep-0008/#id28)

Reply:
PEP8 has a section pertaining to annotations which did not exist at the time.

https://www.python.org/dev/peps/pep-0008/#id52 points 1 and 2 mainly.

https://www.python.org/dev/peps/pep-0484/#arbitrary-argument-lists-and-default-argument-values
has a example of an annotated variable near the end of the section where they use space

Another example (without litteral elllipses) is found here:

https://www.python.org/dev/peps/pep-3107/#syntax

This trips up various linters hence why I fixed it. Additionally, the pep8 (now pycodestyle) tool does not throw any error upon analysis of the file: (Please ignore the other warnings about line length that are set to 120 in the project)

> $ pep8 pose.py
> /usr/lib/python3.6/site-packages/pep8.py:2124: UserWarning: 
> 
> pep8 has been renamed to pycodestyle (GitHub issue #466)
> Use of the pep8 tool will be removed in a future release.
> Please install and use `pycodestyle` instead.
> 
> $ pip install pycodestyle
> $ pycodestyle ...
> 
>   '\n\n'
> pose.py:10:1: E302 expected 2 blank lines, found 1
> pose.py:12:80: E501 line too long (80 > 79 characters)
> pose.py:19:80: E501 line too long (80 > 79 characters)
> pose.py:76:80: E501 line too long (88 > 79 characters)
> pose.py:84:80: E501 line too long (82 > 79 characters)